src/stapi_fastapi/routers/route_names.py

```diff
@@ -0,0 +1,23 @@
+ROOT = "root"
+CONFORMANCE = "conformance"
+LIST_PRODUCTS = "list-products"
+LIST_ORDERS = "list-orders"
+GET_ORDER = "get-order"
+LIST_ORDER_STATUSES = "list-order-statuses"
+LIST_OPPORTUNITY_SEARCH_RECORDS = "list-opportunity-search-records"
+GET_OPPORTUNITY_SEARCH_RECORD = "get-opportunity-search-record"
+
+
+LIST_ORDERS = "list-orders"
+
+# Product
```

Comment:
these are pretty self explanatory, I don't see a need for comments in here. But if, all sections should be commented.

```suggestion
```

Reply:
Done. There were also some duplicates I'd accidentally included.